functions/payment.js

```diff
@@ -1,282 +0,0 @@
-  } catch (err) {
-    throw new functions.https.HttpsError(
-      "invalid-argument",
-      "could not retrieve the event price"
-    );
-  }
-
-  const paymentIntent = await stripe.paymentIntents.create({
-    amount: eventPrice,
-    currency: "gbp",
-    metadata: { integration_check: "accept_a_payment" },
-    receipt_email: context.auth.token.email,
-  });
-
-  await admin.firestore().collection(PURCHASE_TABLE).doc(paymentIntent.id).set({
-    venueId: data.venueId,
-    eventId: data.eventId,
-    userId: context.auth.token.user_id,
-    status: "INITIALIZED",
-  });
-
-  return {
-    client_secret: paymentIntent.client_secret,
-    payment_intent_id: paymentIntent.id,
-  };
-});
-
-exports.setPaymentIntentProcessing = functions.https.onCall(
-  async (data, context) => {
-    authenticationCheck(context);
-
-    if (!(data && data.paymentIntentId)) {
-      throw new functions.https.HttpsError(
-        "invalid-argument",
-        "payment intent data missing"
-      );
-    }
-
-    const documentToUpdate = (
-      await admin
-        .firestore()
-        .collection(PURCHASE_TABLE)
-        .doc(data.paymentIntentId)
-        .get()
-    ).data();
-
-    if (["INITIALIZED", "FAILED"].includes(documentToUpdate.status)) {
-      await admin
-        .firestore()
-        .collection(PURCHASE_TABLE)
-        .doc(data.paymentIntentId)
-        .update({
-          status: "PROCESSING",
-        });
-    }
-
-    return {};
-  }
-);
-
-exports.setPaymentIntentFailed = functions.https.onCall(
-  async (data, context) => {
-    authenticationCheck(context);
-    if (!(data && data.paymentIntentId)) {
-      throw new functions.https.HttpsError(
-        "invalid-argument",
-        "payment intent data missing"
-      );
-    }
-
-    const documentToUpdate = (
-      await admin
-        .firestore()
-        .collection(PURCHASE_TABLE)
-        .doc(data.paymentIntentId)
-        .get()
-    ).data();
-
-    if (documentToUpdate.status === "PROCESSING") {
-      await admin
-        .firestore()
-        .collection(PURCHASE_TABLE)
-        .doc(data.paymentIntentId)
-        .update({
-          status: "FAILED",
-        });
-    }
-
-    return {};
-  }
-);
-
-exports.confirmPaymentIntent = functions.https.onCall(async (data, context) => {
-  authenticationCheck(context);
-  if (!(data && data.paymentIntentId)) {
-    throw new functions.https.HttpsError(
-      "invalid-argument",
-      "payment intent data missing"
-    );
-  }
-
-  const documentToUpdate = (
-    await admin
-      .firestore()
-      .collection(PURCHASE_TABLE)
-      .doc(data.paymentIntentId)
-      .get()
-  ).data();
-
-  if (documentToUpdate.status === "PROCESSING") {
-    await admin
-      .firestore()
-      .collection(PURCHASE_TABLE)
-      .doc(data.paymentIntentId)
-      .update({
-        status: "CONFIRMATION_FROM_STRIPE_PENDING",
-      });
-  }
-
-  return {};
-});
-
-const endpointSecret = STRIPE_CONFIG.endpoint_secret;
```

Comment:
From a quick skim, doesn't look like we've updated the values in `functions/.runtimeconfig.json.example` to account for this change?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40